src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIProjectReferenceFileService.java

```diff
@@ -51,10 +55,8 @@ public UIProjectReferenceFileService(ProjectService projectService, ReferenceFil
 	 * @throws UnsupportedReferenceFileContentError if content is invalid
 	 * @throws IOException if there is an I/O error
 	 */
-	public String addReferenceFileToProject(Long projectId, List<MultipartFile> files, final Locale locale) throws UnsupportedReferenceFileContentError, IOException {
-		Project project = projectService.read(projectId);
-		logger.debug("Adding reference file to project " + projectId);
-
+	public List<UIReferenceFile> addReferenceFileToProject(Long projectId, List<MultipartFile> files, final Locale locale) throws UnsupportedReferenceFileContentError, IOException {
+		List<UIReferenceFile> uiFiles = new ArrayList<>();
```

Comment:
Rename `uiFiles` to `uiReferenceFiles` for clarity

Reply:
Updated in 30597c3a5fc6ba76574a2514eb8720864089a739